pallets/stateful-storage/src/lib.rs

```diff
@@ -451,6 +443,21 @@ pub mod pallet {
 }
 
 impl<T: Config> Pallet<T> {
+	/// Sums the total bytes of each item actions
+	pub fn sum_add_actions_bytes(
```

Comment:
GetCapacityStableWeights uses this function to pass the correct value to the WeightInfo function for `apply_item_actions_with_signature` and `apply_item_actions`.